service/src/main/java/org/cbioportal/service/impl/StaticDataTimestampServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package org.cbioportal.service.impl;
+
+import org.cbioportal.model.TableTimestampPair;
+import org.cbioportal.service.StaticDataTimestampService;
+import org.cbioportal.persistence.StaticDataTimeStampRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class StaticDataTimestampServiceImpl implements StaticDataTimestampService {
+    @Autowired
+    private StaticDataTimeStampRepository staticDataTimeStampRepository;
+
+    @Override
+    public List<TableTimestampPair> getTimestamps(List<String> tables) {
```

Comment:
@Luke-Sikina should we limit this to only gene table and perhaps some others? Not sure if there's anything against being able to get last update time for any table. Can't think of anything atm so maybe it's fine

Reply:
I thought about it and couldn't come up with anything either. For what it's worth, that's part of why I stuck with a hard coded list of tables in the controller. I thought that letting a user request the update date of _any_ table could be problematic.